backend/sirius-web-collaborative-diagrams-api/src/main/java/org/eclipse/sirius/web/collaborative/diagrams/api/IDiagramContext.java

```diff
@@ -35,4 +38,9 @@
 
     List<ViewCreationRequest> getViewCreationRequests();
 
+    Map<UUID, Position> getMovedElementIDToNewPositionMap();
```

Comment:
Why does that have to be an hash map? Unless I am mistaken, you are putting a single element inside. You need a POJO with key/value instead, no need for a whole collection.

Reply:
We have anticipated the move of multiple elements

Reply:
I will write a short ADR for that. Indeed, we just need the moved element id and the new position.

Reply:
see 8b42cc03982374697a6e76d7a2f581e50b44b2a1 for the ADR